From 19096da0d86e64794a9ab0fee189ea3607a6ae92 Mon Sep 17 00:00:00 2001
From: Erez_Zadok <ezk@cs.sunysb.edu>
Date: Mon, 28 May 2007 19:23:42 -0400
Subject: [PATCH] cleanup: remove unnecessary call to inherit_mnt

Our unionfs_lookup doesn't need to call inherit_mnt: the lower mnts will be
correct here (but not if other methods such as unionfs_rename call our
lookup_backend helper function directly).

Signed-off-by: Erez Zadok <ezk@cs.sunysb.edu>
---
 fs/unionfs/inode.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/fs/unionfs/inode.c b/fs/unionfs/inode.c
index dfe516e..85730cc 100644
--- a/fs/unionfs/inode.c
+++ b/fs/unionfs/inode.c
@@ -269,7 +269,6 @@ static struct dentry *unionfs_lookup(struct inode *parent,
 	if (!IS_ERR(ret)) {
 		if (ret)
 			dentry = ret;
-		unionfs_inherit_mnt(dentry);
 	}
 
 	unionfs_check_inode(parent);
-- 
1.5.4.3

